docs/reference/api.yml

```diff
@@ -192,20 +191,6 @@ paths:
           multipart/form-data:
             schema:
               $ref: ./requestBodies/add_or_edit_a_product.yaml
-            examples:
-              example-1:
-                value:
-                  code: '0074570036004'
-                  user_id: myusername
-                  password: mypassword
```

Comment:
we could indeed write on or two simple example here (but you are rightt to remove this one which does not make sense)